getter/get.py

```diff
@@ -406,10 +421,25 @@ def get(args):
 
     elif args.download:
         mkdirs(args.data_dir, False)
-        r = requests.get("https://registry.threesixtygiving.org/data.json")
-        with open("%s/data_original.json" % args.data_dir, "w") as fp:
-            fp.write(r.text)
-        data_all = r.json()
+
+        # Try the registry 5 times to get valid data.json output
+        # This guards against temporary downtime or other issues that the
+        # registry might experience fetching the data.
+        for i in range(0, 5):
+            try:
+                res = session.get(
+                    "https://registry.threesixtygiving.org/data.json"
+                )
+
+                data_all = res.json()
+                if len(data_all) > 0:
+                    with open("%s/data_original.json" % args.data_dir, "w") as fp:
+                        fp.write(res.text)
+                    break
+            except json.JSONDecodeError:
+                print("Warning: Registry JSON data error, retrying in 1 second...")
+                time.wait(1)
```

Comment:
Does `time.wait` exist, do you mean `time.sleep` https://docs.python.org/3.8/library/time.html#time.sleep ?

Reply:
Can confirm
```
>>> import time
>>> time.wait(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'time' has no attribute 'wait'
>>> time.sleep(1)
>>>
```

```suggestion
                time.sleep(1)
```